main.go

```diff
@@ -1,23 +1,40 @@
 package main
 
 import (
+	"flag"
+	"log"
+
 	"github.com/thisissoon/fm-deepmind/fm"
+	"github.com/thisissoon/fm-deepmind/fm_api"
 	"github.com/thisissoon/fm-deepmind/rnd"
-	"log"
 )
 
+var token = flag.String("token", "", "deepmind user")
+var db = flag.String("db", "", "soonfm database string")
+
 func main() {
-	connString := ""
 	data := fm.DataAdapter{}
-	if err := data.Conn(connString); err != nil {
+	if err := data.Conn(db); err != nil {
 		log.Fatal("Scan: %e", err)
 		return
 	}
 
-	gengers := data.GetGenreDataSet(14)
-	d := gengers.Get(rnd.Weight(gengers.GetWeights()))
+	for i := 0; i < 10; i++ {
+		gengers := data.GetGenreDataSet(14)
+		gengersWeights := gengers.GetWeights()
```

Comment:
Can we have parity between var names:

`gengers` should be `genres` or `g`
`gengersWeights` should be `genreWeights` or `weights` or `w`

etc